packages/data/json/9.9.9.json

```diff
@@ -457,7 +457,7 @@
       "questionText": "Is this location a <defined-term>dwelling unit</defined-term>?",
       "questionCodeReference": {
         "displayString": "Vol 2, Div B, Article 9.9.9.1.",
-        "codeNumber": "V2-DB-9.9.9.1"
+        "codeNumber": "9.9.9.1"
```

Comment:
I know this is just in theory, but I had the volume and division in the code number to future proof the lookup for when there may be duplicate part number. Looking at the table of contents again we really only need division since Volume 2 only has Part 9, which Volume 1 doesn't have. Not sure it matters at this point but we could leave the division part in for future purposes.

![Screenshot 2024-07-08 at 1 07 52 PM](https://github.com/bcgov/House-Innovations-HOUSNAV/assets/41478918/ae808645-adcc-4e51-907e-a52115d8f671)

Reply:
Huh.. That's somewhat annoying. lol I thought that references were unique to the Building code but you're right they are not since 1.1. could refer to Division A, B or C

I can add the Volume and Division back. I'll just have to add it to the rest of the references.